catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/history/Historian.java

```diff
@@ -499,6 +506,10 @@ public void setMetacardTypes(List<MetacardType> metacardTypes) {
         this.metacardTypes = metacardTypes;
     }
 
+    public void setSecurity(Security security) {
```

Comment:
Not a fan of methods added solely to support unit testing, but as you've few alternatives, at least drop it to package-private.